internal/services/fluidrelay/fluid_relay_servers_resource.go

```diff
@@ -112,6 +114,16 @@ func (s Server) Attributes() map[string]*pluginsdk.Schema {
 				Type: pluginsdk.TypeString,
 			},
 		},
+
+		"primary_key": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+
+		"secondary_key": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
```

Comment:
These should probably be treated as sensitive properties

```suggestion
		"primary_key": {
			Type:     pluginsdk.TypeString,
			Computed: true,
			Sensitive: true,
		},

		"secondary_key": {
			Type:     pluginsdk.TypeString,
			Computed: true,
			Sensitive: true,
```